src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -66,7 +66,10 @@
     <HelixJobType Condition="'$(FilterToTestTFM)'=='netcore50' and $(TestNugetRuntimeId.Contains('aot'))">test/functional/ilc</HelixJobType>
     <HelixJobType Condition="$(FilterToTestTFM.Contains('net46'))">test/functional/desktop/cli</HelixJobType>
     <HelixJobType Condition="'$(HelixJobType)'=='' And '$(Performance)'=='true'">test/functional/perf/cli</HelixJobType>
-    <HelixJobType Condition="'$(HelixJobType)'==''">test/functional/cli</HelixJobType>
+    <HelixJobType Condition="'$(HelixJobType)'==''">test/functional/cli</HelixJobType>    
+    <HelixSource Condition="'$(HelixSource)'=='' And '$(IsOfficial)'!='' And '$(TestProduct)'!='' And '$(Branch)'!=''">official/$(TestProduct)/$(Branch)</HelixSource>
+    <HelixSource Condition="'$(HelixSource)'=='' And '$(IsOfficial)'=='' And '$(TestProduct)'!='' And '$(Branch)'!=''">pr/$(TestProduct)/$(Branch)</HelixSource>
+    <HelixSource Condition="'$(HelixSource)'!=''">$(HelixSource)</HelixSource>
```

Comment:
This line is a wee bit redundant. :-) Set the thing to itself if it's already set